src/app/terms/page.tsx

```diff
@@ -19,11 +18,15 @@ const termsPageBaseData = generateWebPageStructuredData({
 })
 
 export default function Terms() {
+  const { title } = header
+
   return (
-    <PageLayout>
+    <article>
       <StructuredDataScript structuredData={termsPageBaseData} />
-      <PageHeader title={header.title} description={header.description} />
-      <Content />
-    </PageLayout>
+      <PageHeader.Title>{title}</PageHeader.Title>
+      <section className="prose mt-6">
```

Comment:
So it was just a matter of adding `prose` ?

Reply:
Kinda yeah 😮‍💨🫣🤗